ui/src/app/components/sidenav-view-assessment/sidenav-view-assessment.component.html

```diff
@@ -1,2 +1,101 @@
-<h2>Assessment content coming soon!</h2>
-<button mat-button (click)="closeSidenav()">Close</button>
+<div class="sidenav-view-assessment-container">
+  <div class="sidenav-view-assessment-header">
+    <span class="mat-h2 header-title">Assessment report</span>
+    <button class="close-button" mat-icon-button color="primary" (click)="closeSidenav()">
+      <mat-icon class="close-icon">close</mat-icon>
+    </button>
+  </div>
+  <div class="content">
+    <p>
+      Estimation for {{ srcDbType.toUpperCase() }} to for Spanner conversion:
```

Comment:
Estimation for {{ srcDbType.toUpperCase() }} to Spanner conversion:

Reply:
Updated